app/config/bootstrap/g11n.php

```diff
@@ -10,16 +10,17 @@
  * This bootstrap file contains configurations for all globalizing
  * aspects of your application.
  */
+use lithium\action\Dispatcher as ActionDispatcher;
 use lithium\aop\Filters;
-use lithium\core\Libraries;
```

Comment:
This might be needed when activating the other catalog config. Let's keep this and remove the commented use before catalog.